viewing_party/party.py

```diff
@@ -1,23 +1,213 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+from symbol import single_input
+
+
+def create_movie(title, genre, rating): #Wave 1, step 1 done
+    movie_dict={}
+    if title is None or genre is None or rating is None:
+        return None
+
+    movie_dict["title"]=title
+    movie_dict["genre"]=genre
+    movie_dict["rating"]=rating
+    return movie_dict
+
+def add_to_watched(user_watched_dict,movie_info):
+    user_watched_dict["watched"] = [movie_info]
+    return user_watched_dict

```

Comment:
careful here! You are re-assigning `user_data["watched"]` value, *not* appending *to* the list already inside. 

So, what if this same user added another movie to "watched"? It would overwrite the first movie with the new movie, instead of *appending it* to the list already declared in `user_data["watched"]`